Storybook fails to build if dependant packages rely on `fs`

<!---
Thanks for filing an issue 😄 ! Before you submit, please read the following:

Search open/closed issues before submitting since someone might have asked the same thing before!
-->

# 🙋 Feature Request
Can you please update `yoshi/config/webpack.config.storybook` to include the `node` config returned by the `createCommonWebpackConfig()`

### 😯 Current Behavior

<!--- Tell us what is the current behavior that you would like to see changed -->
If any package relies on `fs`,`net`, or `tsl` node modules, the storybook fails to build with the error:
ERROR in ./node_modules/xy/...
Module not found: Error: Can't resolve 'fs' in 'node_modules/xy/...'      

### 🤔 Expected Behavior

<!--- Tell us how it should work -->
Storybook should build without errors.

### 💁 Possible Solution <!-- optional -->

<!--- Suggest ideas how to implement the addition or change -->
`createCommonWebpackConfig()` already handles this issue by defining the config for `node`:
```
    // https://github.com/webpack/node-libs-browser/tree/master/mock
    node: {
      fs: 'empty',
      net: 'empty',
      tls: 'empty',
      __dirname: true,
    },
```
but the storybook config does not copy that particular part of the config.
Please include the `node` config from the common webpack config into `webpack.config.storybook`.

### 🔦 Context <!-- optional -->

<!--- How has this issue affected you? What are you trying to accomplish?
      Providing context helps us come up with a solution that is most useful in the real world -->

--- a/packages/yoshi/config/webpack.config.storybook.js
+++ b/packages/yoshi/config/webpack.config.storybook.js
@@ -30,5 +30,7 @@ module.exports = config => {
 
   config.plugins = [...(config.plugins || []), new StylableWebpackPlugin()];
 
+  config.node = { ...webpackCommonConfig.node, ...config.node };
+
   return config;
 };

--- a/packages/yoshi/test/webpack-config-storybook.spec.js
+++ b/packages/yoshi/test/webpack-config-storybook.spec.js
@@ -15,6 +15,9 @@ describe('Webpack config storybook', () => {
         extensions: ['.js', '.jsx', '.abc'],
       },
       module: {},
+      node: {
+        net: true,
+      },
     };
   });
 
@@ -46,7 +49,7 @@ describe('Webpack config storybook', () => {
     it('should add all the node lib mock definitions from common config', () => {
       expect(resultConfig.node).to.eql({
         ...commonConfig.node,
-        net: 'empty',
+        net: true,
       });
     });
   });

--- a/packages/yoshi/test/webpack-config-storybook.spec.js
+++ b/packages/yoshi/test/webpack-config-storybook.spec.js
@@ -41,4 +41,13 @@ describe('Webpack config storybook', () => {
       expect(resultConfig.resolve.extensions).to.include('.abc');
     });
   });
+
+  describe('Node lib mocks', () => {
+    it('should add all the node lib mock definitions from common config', () => {
+      expect(resultConfig.node).to.eql({
+        ...commonConfig.node,
+        net: 'empty',
+      });
+    });
+  });
 });

